gtfs2netexfr/README.md

```diff
@@ -8,16 +8,16 @@ format.
 
 ## Installation
 
-To install, you first need to install [PROJ] version 6.3.0.  See [PROJ
-installation instructions].
+To install, you first need to install [PROJ].\
+See [PROJ installation instructions].
 
 [PROJ]: https://proj.org/
-[PROJ installation instructions]: https://github.com/OSGeo/PROJ#installation
+[PROJ installation instructions]: ../README.md#proj-dependency
 
-You also need the following dependencies to be installed.
+You also need the following dependency to be installed.
 
 ```bash
-apt install -y clang libssl-dev
+apt install -y clang
```

Comment:
I would either put that in the PROJ part of the main readme (usefull for any *2netexfr) or completely remove it as it is in makefile.

Reply:
yes I rwill emove it